src/main/scala/lms/core/stub.scala

```diff
@@ -791,13 +797,9 @@ trait LiftPrimitives {
   implicit def chainFloatToRepDouble[A:Manifest](x: A)(implicit c: A => Rep[Float], __pos: SourceContext): Rep[Double] = repFloatToRepDouble(c(x))
 }
 
-/**
- * This file is extremely boilerplate. In fact, most of the code here is copied from a
- * Forge-generated file. We need a static version since Delite (and other projects) depend
- * on it without using Forge.
- */
-trait PrimitiveOps extends Base with OverloadHack {
 
+object PrimitiveTypeLess extends Base {
```

Comment:
this extends Base can be removed

Reply:
indeed, if Rep etc. are not used in PrimitiveTypeLess